### ISSUE TITLE
SyntaxError: Missing parentheses in call to 'print'

### ISSUE THREAD

**Issue by isomorphisms** [ikreymer/cdx-index-client#5]
```
i@scheherezade:/opt/cdx-index-client$ sudo -E -H pip install -r requirements.txt 
[sudo] password for i: 
Requirement already satisfied: requests in /usr/local/lib/python3.5/site-packages (from -r requirements.txt (line 1))
Collecting beautifulsoup (from -r requirements.txt (line 2))
  Using cached BeautifulSoup-3.2.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/user/1001/pip-build-3kgqmdp4/beautifulsoup/setup.py", line 22
        print "Unit tests have failed!"
                                      ^
    SyntaxError: Missing parentheses in call to 'print'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/user/1001/pip-build-3kgqmdp4/beautifulsoup/
```

**Comment by bhavyaagg** [comment]
You are using Python3 which is not compatible with beautifulsoup3. If you still want to use it, you can go for beautifulsoup4.

**Comment by agnelvishal** [comment]
@ikreymer Close this issue?

**Comment by ikreymer** [comment]
Yep! Merged PR #8 which removed BeautifulSoup altogether, and #10 adds support for python 3.
